### ISSUE TITLE
Docker Script Run Issue

### ISSUE THREAD

**Issue by sweber15** [UBC-MDS/DSCI522_group315#42]
Hi team, 

I tried running the DockerScript and got an error:

Input:
`docker run -it --rm -v C:\Users\sarah\Documents\DSCI_Labs\Term_4\DSCI_522\DSCI522_group315:/root/ufc dsci-522-ufc cd root/ufc make all
`

Output:
`Unable to find image 'dsci-522-ufc:latest' locally
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: pull access denied for dsci-522-ufc, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'C:\Program Files\Docker\Docker\Resources\bin\docker.exe run --help'.`

Any ideas of what is wrong?

**Comment by SamEdwardes** [comment]
I think I figured out the issue... the problem is that we have not uploaded a version of this image to docker hub yet. So it is looking on your computer for image `dsci-522-ufc`, realizes it does not exist, then attempts to search on docker hub for it (where it does not exist).

As a workaround, first run:

`docker build --tag dsci-522-ufc .`

Then run your input from above. I think that should work.

Once we have finalized the image we can upload to docker hub to avoid this issue in the future.
